project/src/main/webapp/script.js

```diff
@@ -12,17 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/*** Side Navigation Menu ***/
+/* Set the width of the side navigation to 250px and the left margin of the page content to 250px */
+function openNav() {
+  document.getElementById("side-menu").style.width = "250px";
+  document.getElementById("main").style.marginLeft = "250px";
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* Set the width of the side navigation to 0 and the left margin of the page content to 0 */
+function closeNav() {
+  document.getElementById("side-menu").style.width = "0";
+  document.getElementById("main").style.marginLeft = "0";
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/*** Dark Mode Toggle ***/
+function darkMode() {
+  var element = document.body;
```

Comment:
It's best not to use `var` (it has some issues). When you need a name to actually point to different objects/values at different times (like the index variable of a `for` loop), use `let`. If you're just assigning it and then not changing what it refers to (like in this case), use `const`.